cfgov/regulations3k/models/django.py

```diff
@@ -148,7 +159,10 @@ class Meta:
 
 
 class Subpart(models.Model):
-    label = models.CharField(max_length=255, blank=True)
+    label = models.CharField(
+        max_length=255,
+        validators=[validate_label],
```

Comment:
How about we add some help text, while we're at it? Try and stave off showing any validation errors in the first place.

```suggestion
        validators=[validate_label],
        help_text='Labels always require at least 1 alphanumeric character, '
                  'then any number of alphanumeric characters and hyphens.',
```

(sorry, this'll require recreating the migration)